packages/react-instantsearch-hooks-dom/src/ui/Pagination.tsx

```diff
@@ -5,6 +5,49 @@ import { isModifierClick } from './lib/isModifierClick';
 
 import type { CreateURL } from 'instantsearch.js';
 
+export type PaginationTranslations = {
+  /**
+   * The label for the first page's button.
+   */
+  first: string;
+  /**
+   * The label for the previous page's button.
+   */
+  previous: string;
+  /**
+   * The label for the next page's button.
+   */
+  next: string;
+  /**
+   * The label for the last page's button.
+   */
+  last: string;
+  /**
+   * The label for a page's button.
+   */
+  page(currentPage: number): string;
+  /**
+   * The accessible label for the first page's button.
+   */
+  ariaFirst: string;
+  /**
+   * The accessible label for the previous page's button.
+   */
+  ariaPrevious: string;
+  /**
+   * The accessible label for the next page's button.
+   */
+  ariaNext: string;
+  /**
+   * The accessible label for the last page's button.
+   */
+  ariaLast: string;
+  /**
+   * The accessible label for a page's button.
+   */
+  ariaPage(currentPage: number): string;
+};
```

Comment:
Following https://github.com/algolia/instantsearch-specs/pull/120, should we have a `ariaCurrent` string for the current page accessible label? Or maybe `ariaPage()` could accept a `isCurrent` flag to compose the string differently.